README.md

```diff
@@ -35,22 +35,14 @@ To set up environment variables for the project, follow these steps:
 
 Note: These variables are currently only used for end-to-end tests.
 
-To start a dev server running all the modules simultaneously, run:
-
-```bash
-yarn start
-```
-
-This command uses the [openmrs](https://www.npmjs.com/package/openmrs) tooling to fire up a dev server running `esm-patient-chart` as well as the specified module.
-
-Note that this is very resource-intensive.
-
 To start a dev server for a specific module, run:
 
 ```bash
 yarn start --sources 'packages/esm-<insert-package-name>-app'
 ```
 
+This command uses the [openmrs](https://www.npmjs.com/package/openmrs) tooling to fire up a dev server running `esm-patient-chart` as well as the specified module.
```

Comment:
Here the name will be esm-patient-chart or esm-patient-management ?

Reply:
It should be `esm-patient-management`

Reply:
Changed it. Please let me know if anything more need to be updated.